recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake, tools
+import glob
+import os
+
+class CubicInterpolationConan(ConanFile):
+    name = "CubicInterpolation"
```

Comment:
```suggestion
    name = "cubic-interpolation"
```

CamelCase is not allowed. All characters must be lower-cased.